resources/content/schema/base/clusterHostMap.json

```diff
@@ -0,0 +1,3 @@
+{
+    "collectionMethods" : []
```

Comment:
If you don't want this object to show up in `/v1` just de-authorize the type, as opposed to limiting the collection methods.  By nothing having things in `/v1` people get the false impression that the object is not accessible, but all it really means is that you can't list them.  One could still fetch the object by ID if they knew it.